pkg/apiserver/request/requestinfo.go

```diff
@@ -204,7 +204,22 @@ func (r *RequestInfoResolver) GetRequestInfo(req *http.Request) (RequestInfo, er
 		requestInfo.Verb = "deletecollection"
 	}
 
-	return requestInfo, nil
+	return &requestInfo, nil
+}
+
+type requestInfoKeyType int
+
+const requestInfoKey requestInfoKeyType = iota
```

Comment:
It's what the go context package suggests: https://github.com/golang/net/blob/master/context/context.go#L100